indico/modules/events/abstracts/forms.py

```diff
@@ -74,6 +74,29 @@ def validate_contrib_type_required(self, field):
             raise ValidationError(_('The event has no contribution types defined.'))
 
 
+class AbstractReviewingSettingsForm(IndicoForm):
+    """Settings form for abstract reviewing"""
+
+    num_answers = IntegerField(_('Number of choices'), [NumberRange(min=2), DataRequired()],
+                               description=_('The number of choices reviewers have for each question.'))
+    scale_lower = IntegerField(_('Scale (from)'), [InputRequired()])
```

Comment:
How often are these used? Fiddling with the number of choices and scale will cause the values of the choices to be fractional unless carefully adjusted. Maybe we can hide this complexity with a `custom scale` checkbox. If not checked, the scale adjusts so that the values of the choices start at **1**, go up to `num_choices`, and have increments of **1**.